### PR TITLE
Added create project in organizations

### PR DESCRIPTION
**Pull request by SeqLaz**
Added the section for creating and/or transferring projects in organizations

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-2695 Document create a project in QFC organization](https://app.clickup.com/t/2192114/QF-2695)

**Comment by lucienicolier** [comment]
@SeqLaz can you make the changes and I will recheck? Thanks

**Review by SeqLaz — COMMENTED** [review]
Yes the improvement in the text and layout are perfect

**Comment by lucienicolier** [comment]
@SeqLaz I made a commit to fix the list, there was only step 1.

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 1]
```diff
@@ -153,6 +153,62 @@ A form will ask you for project name, description and local directory. In the lo
...
+
+    ![Pushing changes to QFieldCloud](../../assets/images/project_organization_09_pushing_changes_to_cloud.png)
+
+10. You can verify that the files are present in the Organization project.
+
+    ![Project files over view in QFieldCloud](../../assets/images/project_organization_10_files_overview_in_cloud.png)
+
+**Option 2: Moving the project from your own account to the organizations:**
+
+1. If you already have a project in QFieldCloud (Refer to [configure your cloud project](#create-and-configure-your-cloud-project)). in the project click on "Settings" and select "Transfer ownership of this project" to choose the desired Organization for the transfer.
```
[the remark this answers is not in the document]
Comment (SeqLaz) [reply documentation/get-started/tutorials/get-started-qfc.en.md:204]: Perfect

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 2]
```diff
@@ -153,6 +153,62 @@ A form will ask you for project name, description and local directory. In the lo
...
+
+    ![Transferring to Organization](../../assets/images/project_organization_11_transfering_to_organization.png)
+
+2. A pop-up window will appear to confirm the transfer. In order to proceed, you will need to type "Here be dragons" and click "Transfer project".
```
[the remark this answers is not in the document]
Comment (SeqLaz) [reply documentation/get-started/tutorials/get-started-qfc.en.md:208]: Made it

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 3]
```diff
@@ -153,6 +153,62 @@ A form will ask you for project name, description and local directory. In the lo
...
+
+    ![Creating Project](../../assets/images/project_organization_02_creating_project.png)
+
+3. Select "Create a new empty project".
+
+    ![New empty project](../../assets/images/project_organization_03_new_empty_project.png)
+
+4. On QGIS in QFieldSync, you will see the new project listed.
+
+    ![New project created](../../assets/images/project_organization_04_new_project_created.png)
```
[the remark this answers is not in the document]
Comment (SeqLaz) [reply documentation/get-started/tutorials/get-started-qfc.en.md:176]: Yes, the image is an overview to the new project created in QFieldCloud

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 4]
```diff
@@ -113,6 +113,78 @@ By double-clicking on a project in the list, you can see and edit the specific p
...
+
+9.  Finally, push the changes to the cloud.
+
+![Pushing changes to QFieldCloud](../../assets/images/project_organization_09_pushing_changes_to_cloud.png)
+
+10. You can verify that the files are present in the Organization project.
+
+![Project files over view in QFieldCloud](../../assets/images/project_organization_10_files_overview_in_cloud.png)
+
+**Option 2: Moving the project from your own account to the organizations:**
```
[the remark this answers is not in the document]
Comment (SeqLaz) [reply documentation/get-started/tutorials/get-started-qfc.en.md]: Made it, its fixed

File: documentation/get-started/tutorials/get-started-qfc.en.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (lucienicolier) [reply documentation/get-started/tutorials/get-started-qfc.en.md:176]: So maybe add a comment on this image and add the step 4 under

File: documentation/get-started/tutorials/get-started-qfc.en.md
[the remark this answers is not in the document]
Comment (SeqLaz) [reply documentation/get-started/tutorials/get-started-qfc.en.md:1]: Done

File: documentation/get-started/tutorials/get-started-qfc.en.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (SeqLaz) [reply documentation/get-started/tutorials/get-started-qfc.en.md:176]: It's fixed now

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 5]
```diff
@@ -153,6 +153,62 @@ A form will ask you for project name, description and local directory. In the lo
...
+
+2. Once you get into the organization, click on "Create a project".
+
+    ![Creating Project](../../assets/images/project_organization_02_creating_project.png)
+
+3. Select "Create a new empty project".
+
+    ![New empty project](../../assets/images/project_organization_03_new_empty_project.png)
+
+4. And you willl see in the overview of the new project created.
```
Comment (lucienicolier) [inline documentation/get-started/tutorials/get-started-qfc.en.md:174]:
```suggestion
4. You can see the new project in the overview.
```

**Review by lucienicolier — APPROVED** [review]

**Pull request merged**